bundles/com.zeligsoft.domain.dds4ccm/src/com/zeligsoft/domain/dds4ccm/utils/DDS4CCMMigrationUtil.java

```diff
@@ -323,8 +326,9 @@ public static String getModelVersion(final Model model) {
 	 */
 	public static void addMigrationAnnotation(final Model model,
 			final String toVersion) throws Exception {
+		TransactionalEditingDomain editingDomain = TransactionUtil.getEditingDomain(model);
```

Comment:
Can you please convert this command to an emf command? For example, RecordingCommand. I realized that this is a DDS4CCM plugin where we should avoid the dependency on UI plugins. I would rather avoid the GMF plugin if possible.